custom_image/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Custom Image
+============
+
+This module intents to add images to any object, or to override the standard Odoo behavior on existing ones.
```

Comment:
to achieve what?
I'll put review on hold until you clarify what are the main goals and - for instance - how is this different from base_multi_image and/or if it's compatible w/ it :wink: 

A real use case might help